libs/cardano-ledger-binary/cardano-ledger-binary.cabal

```
@@ -57,8 +57,8 @@ library
     bytestring,
     cardano-base >=0.1.1.0,
     cardano-binary >=1.7,
-    cardano-crypto-class ^>=2.2,
-    cardano-crypto-praos >=2.2,
+    cardano-crypto-class ^>=2.2.4,
```

Using `^>=` will make it overly restrictive and will not let it pick `2.2.5` if it is ever released.
```suggestion
    cardano-crypto-class >=2.2.4 && <2.3,
```